.github/workflows/dev-deploy.yml

```diff
@@ -0,0 +1,81 @@
+name: Dev branch deploy
+
+on:
+  push:
+    branches:
+      - dev
```

Comment:
Hi, thanks for working on this. Can we please not run this on every commit as this would be very resource intensive. I think once a week or something similar would be best for this action.

Reply:
Sorry for the late reply. I have make an update, add a job to ensure that the interval between image publishes is greater than a week. 
And there is a question: is `DOCKER_PASSWORD` an Docker Hub token? Although it doesn't matter even if it's ineffective, I'm just using it to avoid the rate limit of API requests.
Thanks!

Reply:
Hi, no worries! Can we setup it like a separate action and do the schedule part as mentioned in (Action docs)[https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule] as it seems much easier.

Reply:
'docker/metadata-action' and 'docker/build-push-action' use git ref of the event that triggers the workflow to automate a lot of things, changing to schedule, we need to configure it in a more complex way.
Or we can just delete `-H "Authorization: Bearer ${{ secrets.DOCKER_PASSWORD }}"` in curl command, our use case doesn't cause rate limiting.